db/services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids, actors_ids):
+    queryset = Movie.objects.all()
+    if genres_ids is None and actors_ids is None:
+        return queryset
+    if genres_ids is not None and actors_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+        queryset = queryset.filter(actors__id__in=actors_ids)

```

Comment:
you can write few operands in one line, for example: filter(genres__id__in=genres_ids, actors__id__in=actors_ids)